src/Data/Bin/Properties.agda

```diff
@@ -364,25 +367,22 @@ fromℕ-homo-+ (ℕ.suc m) n = begin
 +-isSemigroup : IsSemigroup _+_
 +-isSemigroup = record
   { isMagma = +-isMagma
-  ; assoc = +-assoc
+  ; assoc   = +-assoc
   }
 
-+-0-isMonoid : IsMonoid _+_ 0B
++-0-isMonoid : IsMonoid _+_ zero
```

Comment:
Please could you put `0B` back?

Reply:
Done

Reply:
Thanks, but I think you missed out `+-0-isCommutativeMonoid` and `+-0-monoid`.

Reply:
Done.

Reply:
This change isn't appearing.